Automattic/wp-calypso
# BackupBrowser: release granular restore for single files

Related to https://github.com/Automattic/jetpack-backup-team/issues/128

## Proposed Changes

* Add `calypso_jetpack_backup_browser_restore_single_file ` event to track when user clicks on `Restore` button on a single file in the backup browser.
* Add `calypso_jetpack_granular_restore_confirm` event to track when user clicks on `Confirm restore` in the granular restore confirmation page.
* Add `calypso_jetpack_granular_restore_cancel` event to track when user clicks on `Cancel` in the granular restore confirmation page.
* Add `calypso_jetpack_granular_restore_goback` event to track when a user clicks on `Go Back` in the granular restore confirmation page.
* Enable restore button by removing feature flag condition. 

## Testing Instructions
* Start a Jetpack Cloud or Calypso Live branch.
* Pick a file and ensure the `Restore` button is visible (except on the WordPress item).
* Open developer console, on network tab.
* Click on `Restore` and it should redirect you to `/backup/:site/granular-restore/:rewindId`.
* You should see a `t.gif` call in the developer console, with the event `calypso_jetpack_backup_browser_restore_single_file `.
* It should redirect you to the granular restore confirmation page.
* Validate that you see a `t.gif` call with the event `calypso_jetpack_granular_restore_cancel` when you click on `Cancel`.
* Validate that you see a `t.gif` call with the event `calypso_jetpack_granular_restore_goback` when you click on `<- Go Back`.
* Validate that you see a `t.gif` call with the event `calypso_jetpack_granular_restore_confirm` when you click on `Confirm restore`.

## Pre-merge Checklist
